addons/mail/static/src/core/web/recipient_list.js

```diff
@@ -22,4 +24,11 @@ export class RecipientList extends Component {
             }
         });
     }
+
+    getRecipientText(recipient) {
+        return escapeHTML(
```

Comment:
`escapeHTML` is not necessary when using inside t-esc, or t-out (without markup)